src/styles/blocks/contact.scss

```
@@ -0,0 +1,175 @@
+.contact-us {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  gap: 40px;
+
+  &__title {
+    font-size: 48px;
+    line-height: 48px;
+    font-weight: 700;
+    letter-spacing: -1.5px;
+    margin: 0 auto 58px;
+    position: relative;
+    color: #000;
```

```suggestion
  &__title {
    font-size: 48px;
    line-height: 48px;
    font-weight: 700;
    letter-spacing: -1.5px;
    margin: 0 auto 58px;
    position: relative;
    color: #000;
```
You are repeating these styles three times (here, in 'testimonials' and in 'gallery'). Why don't you create a mix of classes like `<section class="contact-us section">` and make an inside heading with class `section__heading` and give all neede styles only once?